Makefile

```diff
@@ -3,6 +3,9 @@ SHELL := /bin/bash
 MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 GO ?= go
+KUADRANT_NAMESPACE=kuadrant-system
+GIT_HASH := $(shell git rev-parse --short=7 HEAD)
```

Comment:
These are executed for every make target. Maybe better to compute those only on `install` target.

Maybe something like:

```make
install: fmt vet
    $(eval GIT_HASH  := $(shell git rev-parse --short=7 HEAD))
    GOBIN=$(PROJECT_PATH)/bin $(GO) install -ldflags "-X 'github.com/kuadrant/kuadrantctl/version.GitDirty=$(GIT_DIRTY)' -X 'github.com/kuadrant/kuadrantctl/version.GitHash=$(GIT_HASH)'"
```
```

Reply:
Sure, sounds good. Will do...